iOS/UI/Shared/ReplySummaryView.swift

```diff
@@ -214,33 +211,10 @@ class ReplySummaryView: BaseView {
         self.promptLabel.match(.left, to: .right, of: self.lineDotView, offset: .standard)
         self.promptLabel.pin(.bottom, offset: .custom(8))
         
-        self.counter.sizeToFit()
-        
-        if self.promptLabel.isVisible {
-            self.counter.match(.left, to: .right, of: self.promptLabel, offset: .standard)
-        } else {
-            self.counter.match(.left, to: .right, of: self.lineDotView, offset: .standard)
-        }
-        self.counter.pin(.bottom, offset: .custom(8))
-        
-        self.promptButton.height = 30
-        self.promptButton.width = self.width
-        self.promptButton.left = self.lineDotView.left
-        self.promptButton.pin(.bottom)
+        self.promptButton.expandToSuperviewSize()
         
         self.lineDotView.expandToSuperviewHeight()
     }
-
-    private func subscribeToUpdates() {
-        self.cancellables.forEach { cancellable in
-            cancellable.cancel()
-        }
-        
-        self.controller?.repliesChangesPublisher.mainSink { [unowned self] _ in
```

Comment:
The view is already getting these updates so removing this fixed an extra layout call.